wiki/People/Beatmap_Nominators/Rules/en.md

```diff
@@ -8,46 +8,68 @@ Beatmap Nominators possess the tools to nominate beatmaps and shape the world of
++ **Do not nominate a beatmap without properly checking all difficulties of the beatmap all by yourself.** If you are unfamiliar with the game mode some of the difficulties are mapped in, ask fellow Beatmap Nominators for help. Improper checking includes just running a modding tool before nominating or not even downloading the beatmap.
 + **Do not qualify beatmaps of modes which you are not a full member of, unless they were nominated by someone who is.** This ensures that a full nominator is always part of the process for each mode.
 + **Hybrid beatmapsets must be nominated by full nominators from two of its modes in order to be qualified**, as well as gain clear approval from any remaining modes' nominators in the modding discussion. This means only full members may nominate hybrid sets, and ensures that a full nominator takes responsibility for each mode (regardless of nominating or just giving approval in the discussion). In this scenario, nominators proficient in multiple modes can approve more than one mode.
 + **Do not nominate beatmaps of game modes you are not formally qualified for.** You must be qualified to nominate at least one of the game modes included in the mapset by being a member of the respective part of the Beatmap Nominators in order to be allowed to nominate it.
-+ **Do not veto beatmaps of game modes you are not formally qualified for.** You are not allowed to nominate them so blocking a nomination without the ability to renominate the beatmap makes no sense. This does not apply to vetoes regarding audio, backgrounds, and other content that does not directly pertain to the mapset's game mode.
-+ **Do not nominate maps with unrankable issues.** The General Ranking Criteria and additionally all mode-specific Ranking Criteria apply to all sets with mode-specific difficulties.
-+ **Do not qualify mapsets whose song already has a Qualified mapset, unless no game modes of both sets overlap - this includes remixed versions**. For example an osu!taiko and osu! mapset of the same song can exist in the Qualified section, whilst an osu!taiko and osu!/osu!taiko hybrid set cannot. This ensures variety among Ranked beatmaps within their respective game modes.
-+ **If a bubbled map contains unrankable issues, its bubble must be popped.** This applies even if you do not intend to nominate the mapset.
+
+
+### Forum Nomination
+
++ **Beatmaps qualified in modding v1 that need to be disqualified must be brought to the attention of the NAT.** 
++ **__Every__ mapset is approved through two nominations.** Other factors such as the mapset itself being a Marathon, a normal spread or its length do not matter, the amount of nominations required in order to move it to the Qualified section remains at two. More nominations than necessary must not be done.
```

Comment:
Italics not used anywhere else, so it can be removed (it was broken anyway).
```suggestion
+ **Every mapset is approved through two nominations.** Other factors such as the mapset itself being a Marathon, a normal spread or its length do not matter, the amount of nominations required in order to move it to the Qualified section remains at two. More nominations than necessary must not be done.
```